src/puya/awst/nodes.py

```diff
@@ -228,22 +284,33 @@ def accept(self, visitor: StatementVisitor[T]) -> T:
 
 @attrs.frozen
 class LoopExit(Statement):
-    """break out of the current innermost loop"""
+    """
+    Goto the statement immediately following the current innermost loop construct
+    """
 
     def accept(self, visitor: StatementVisitor[T]) -> T:
         return visitor.visit_loop_exit(self)
 
 
 @attrs.frozen
 class LoopContinue(Statement):
-    """continue with the next iteration of the current innermost loop"""
+    """
+    Goto the end of the loop_body of the current innermost loop construct and continue to the next
+    iteration of the loop.
+    """
 
     def accept(self, visitor: StatementVisitor[T]) -> T:
         return visitor.visit_loop_continue(self)
 
 
 @attrs.frozen
 class ReturnStatement(Statement):
+    """
+    Return from the current subroutine with an optional value.
```

Comment:
Maybe add that the value is only optional if the subroutine is typed as returning void.